spec/index.html

```diff
@@ -440,14 +440,14 @@ <h3>Quoted Literals</h3>
         <a href="#grammar-production-RDFLiteral">RDFLiteral</a>) have a lexical form
         followed by a language tag, a datatype IRI, or neither.
         The representation of the lexical form consists of an initial delimiter
-        (e.g., <code>"</code> (<span class="codepoint">U+0022</span>));
+        (e.g., <code>&quot;</code> (quotation mark, code point <code class="codepoint">U+0022</code>));
```

Comment:
A question of consistency... This might be rejected, but it should at least be considered.
```suggestion
        (e.g., '<code>&quot;</code>' (quotation mark, code point <code class="codepoint">U+0022</code>));
```